<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <widget class="QPushButton" name="play">
         <property name="text">
          <string>Play</string>
         </property>
         <property name="icon">
          <iconset resource="repository.qrc">
           <normaloff>:/new/prefix1/Icons/022-video.png</normaloff>:/new/prefix1/Icons/022-video.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="directory">
         <property name="text">
          <string>Dir</string>
         </property>
         <property name="icon">
          <iconset resource="repository.qrc">
           <normaloff>:/new/prefix1/Icons/015-folder.png</normaloff>:/new/prefix1/Icons/015-folder.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>1x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2x</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="maximumSize">
          <size>
           <width>35645</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="endTime">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="volume">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="7">
        <widget class="QSlider" name="horizontalSlider">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="repository.qrc"/>
 </resources>
 <connections/>
</ui>
